ci_scripts/utils/aws_helpers.py

```diff
@@ -0,0 +1,74 @@
+import boto3
+import logging
+import os
+import mimetypes
+
+BUCKET = 'mbed-target'
+
+logger = logging.getLogger(__name__)
+
+S3_REGION = os.getenv('AWS_DEFAULT_REGION', 'eu-west-2')
+S3_CONFIG = {
+    "aws_access_key_id": os.getenv('AWS_ACCESS_KEY_ID'),
+    "aws_secret_access_key": os.getenv('AWS_SECRET_ACCESS_KEY'),
+}
+
+
+def upload_file(file: str, bucket_dir: str):

```

Comment:
In this file it would be good to have some hints about why this is being done, knowing nothing about AWS I'm a bit lost here.